adminapp/src/components/EligibilityConstraints.jsx

```diff
@@ -0,0 +1,149 @@
+import api from "../api";
```

Comment:
Nothing changed in this component except for the prop names. It was taken from the OfferingDetailPage and made into its own component.